mlflow/keras.py

```diff
@@ -217,6 +217,9 @@ def _is_tf_keras(model):
     # save keras model to path/data/model.h5
     model_subpath = os.path.join(data_subpath, _MODEL_SAVE_PATH)
     model_path = os.path.join(path, model_subpath)
+    # if the path ends with ".h5", save will use hdf5
+    if "save_format" in kwargs and kwargs["save_format"] == "tf" and model_path.endswith(".tf"):
```

Comment:
Since `_MODEL_SAVE_PATH` wasn't modified, the model path will actually end with `.h5` and never `.tf`, so the body of this if statement can never be triggered, right?